src/config/dynamic-theme-fixes.config

```
@@ -1552,6 +1552,12 @@ INVERT
 .fiu-header__branding
 .fiu-sample-list__item img
 .fiu-gallery-head__text img
+================================
```

I accidentally an entire `/n` 🤦‍♂️ 

```suggestion

================================
```